dll/win32/getuname/lang/ro-RO.rc

```diff
@@ -3,7 +3,8 @@
  * FILE:            dll\win32\getuname\lang\ro-RO.rc
  * PURPOSE:         Unicode Description Strings
  * PROGRAMMERS:     Baruch Rutman (peterooch at gmail dot com)
- * TRANSLATOR:      Ștefan Fulea (stefan dot fulea at mail dot com)
+ * TRANSLATORS:     Ștefan Fulea (stefan dot fulea at mail dot com)
+ *                  Andrei Miloiu (miloiuandrei@gmail.com)
```

Comment:
You may also add the years of contribution (say, the way you did it [here](https://github.com/reactos/reactos/blob/master/dll/cpl/input/lang/ro-RO.rc#L6)).